console/src/main/scripts/plugins/accounts/ts/accountsGlobals.ts

```diff
@@ -63,4 +93,28 @@ module HawkularAccounts {
     warning(message: string): void;
     error(message: string): void;
   }
+
+  export class Invitation implements IInvitation {
+    id:string;
+    token:string;
+    email:string;
+    organization:HawkularAccounts.IOrganization;
+    role:HawkularAccounts.IRole;
+    acceptedAt:Date;
+
+    constructor(email:string, role:HawkularAccounts.IRole) {
+      this.email = email;
+      this.role = role;
+    }
+  }
+
+  export class Role implements IRole {
```

Comment:
Shouldn't be any need to duplicate. Interfaces can extend other interfaces and classes can even have 'abstract' classes now.